pkg/api/controller.go

```diff
@@ -1515,6 +1504,11 @@ func handleAPIError(w http.ResponseWriter, err error) bool {
 		errors.Is(err, model.ErrValidationError):
 		writeError(w, http.StatusBadRequest, err)
 
+	case errors.Is(err, graveler.ErrBranchExists),
+		errors.Is(err, graveler.ErrTagAlreadyExists),
+		errors.Is(err, graveler.ErrNotUnique):
```

Comment:
Might be more readable if `ErrBranchExists` and `ErrTagAlreadyExists` were both wrapping `graveler.ErrNotUnique` - could make this check a little more generic.

Reply:
Done.